nilearn/maskers/base_masker.py

```diff
@@ -424,3 +429,105 @@ def __sklearn_tags__(self):
             surf_img=True, niimg_like=False, masker=True
         )
         return tags
+
+    def transform(self, imgs, confounds=None, sample_mask=None):
+        """Apply mask, spatial and temporal preprocessing.
+
+        Parameters
+        ----------
+        imgs : 1D/2D Surface image
+            Images to process.
+
+        confounds : CSV file or array-like, default=None
+            This parameter is passed to signal.clean. Please see the related
+            documentation for details.
```

Comment:
Is it possible to have a link to the related documentation from here?

Reply:
add a link, but this gets repeated across all maskers so probably worth refactoring in a follow up